pipelines/LSSTComCam/DRP_InjectedPVIDiffim.yaml

```diff
@@ -0,0 +1,345 @@
+description: DRP Pipeline for LSSTComCam with injected point sources
+instrument: lsst.obs.lsst.LsstComCam
+
+imports:
+  - location: $DRP_PIPE_DIR/pipelines/LSSTComCam/DRP.yaml
+
+parameters:
+  fakesType: 'injected_'
+  injection_prefix: 'injection_'
+  injected_prefix: 'injected_'
+
+tasks:
+  inject_visit:
+    class: lsst.source.injection.inject_visit.VisitInjectTask
+    config:
+      external_psf: false
+      external_photo_calib: false
+      external_wcs: false
+      connections.input_exposure: pvi
+      connections.output_exposure: injected_pvi
+      connections.output_catalog: injected_pvi_catalog
+      selection: "np.isin(injection_catalog['visit'], {visit})"
```

Comment:
Can I ask why you've decided to hard-code the `VisitInjectTask` here, rather than dynamically create it as we do with other injection pipelines? Putting this in would be the second instance in the stack (after `pipelines/_ingredients/ApPipeWithFakes.yaml`).

Reply:
Because dynamically creating this has a key shortcomings for our case, as it doesn't include the matching tasks, with the associated metrics tasks to verify that we have found our injections

Reply:
From `make_injection_pipeline --help`:
```
  -a, --additional-pipelines FILE [FILE ...]
      Location(s) of additional input pipeline definition YAML file(s).Tasks
      from these additional pipelines will be added to the output injection
      pipeline.
```
This allows you to define the additional task stubs you need in their own stand-along YAML, and then build a complete SI YAML when `drp_pipe` gets constructed.